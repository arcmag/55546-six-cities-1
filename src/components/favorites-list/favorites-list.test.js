import renderer from 'react-test-renderer';

import FavoritesList from './favorites-list';

const mock = {
  cities: []
};

describe(`Test FavoritesList`, () => {
  it(`test renderer`, () => {
    const tree = renderer.create(<FavoritesList
      setActionCard={jest.fn()}
      clearActionCard={jest.fn()}
      offersCities={mock.cities}
    />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
