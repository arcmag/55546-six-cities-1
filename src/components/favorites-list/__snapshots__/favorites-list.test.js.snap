// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test FavoritesList test renderer 1`] = `
<ul
  className="favorites__list"
/>
`;
